elastic/beats
# [Packetbeat] Add AF_PACKET metrics

## What does this PR do?

- Add metrics for AF_PACKET interfaces

## Why is it important?

Provides information about the health of af_packet interfaces.

## Checklist


## How to test this PR locally

- Run Packetbeat on Linux with an AF_PACKET interface
- Ensure that metrics http endpoint is enabled.
- `curl -sL http://localhost:5066/inputs | jq '.[] | select(.input=="af_packet")'`

## Related issues

- Closes #35428

## Logs

```json
{
  "device": "enp0s3",
  "id": "4B566DB361E581EE-af_packet::enp0s3",
  "input": "af_packet",
  "packets": 799,
  "polls": 211,
  "socket_drops": 2,
  "socket_packets": 801,
  "socket_queue_freezes": 166
}
```
